Bencodetest.cpp

```diff
@@ -0,0 +1,93 @@
+#include "Bencoding.hpp"
+#include "TorrentFileParser.hpp"
+#include "Tracker.hpp"
+
+int main() {
+	//std::string stringtest = "3:hogAB";
```

Comment:
I suggest writing some unit tests using a framework: maybe https://en.wikipedia.org/wiki/Google_Test? (Disclaimer: I haven't actually tried it)